tools/lsp/preview.rs

```diff
@@ -1331,6 +1333,9 @@ async fn parse_source(
     cc.include_paths = include_paths;
     cc.library_paths = library_paths;
 
+    cc.debug_info = true;
```

Comment:
Is debug info not something else?  Do we need it?

Reply:
That is what enables the `debug` in Elements. We do need that, so we should explicitly set it.